src/prepare.rs

```diff
@@ -69,11 +69,17 @@ impl<'a> Prepare<'a> {
         Ok(())
     }
 
-    fn remove_cargo_config(&self) -> Result<(), Error> {
-        let path = self.source_dir.join(".cargo").join("config");
-        if path.exists() {
-            crate::utils::remove_file(&path)?;
-            info!("removed {}", path.display());
+    fn remove_override_files(&self) -> Result<(), Error> {
+        let paths = [
+            &Path::new(".cargo").join("config"),
+            Path::new("rust-toolchain"),
```

Comment:
```suggestion
            &Path::new(".cargo").join("config"),
            &Path::new(".cargo").join("config.toml"),
            Path::new("rust-toolchain"),
            Path::new("rust-toolchain.toml"),
```

any other synonyms I'm missing? I personally keep trying `.rust-toolchain` but that isn't supported.

Reply:
I think that should be it - https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure says it walks parent directories, but we always build crates on their own, not in a workspace.